config/Development.toml

```diff
@@ -103,9 +103,15 @@ base_url = "https://apis.sandbox.globalpay.com/ucp/"
 [connectors.worldline]
 base_url = "https://eu.sandbox.api-ingenico.com/"
 
+[connectors.forte]
+base_url = "https://sandbox.forte.net/api/v3/"
+
 [scheduler]
 stream = "SCHEDULER_STREAM"
 
 [scheduler.consumer]
 disabled = false
 consumer_group = "SCHEDULER_GROUP"
+
+[secrets]
```

Comment:
remove this secrets block unless it is necessary for your connector

Reply:
done